fastlane/lib/fastlane/runner.rb

```diff
@@ -109,7 +109,9 @@ def find_alias(action_name)
 
     # This is being called from `method_missing` from the Fastfile
     # It's also used when an action is called from another action
-    def trigger_action_by_name(method_sym, custom_dir, *arguments)
+    # @param from_action Indicates if this action is being trigged by another action.
+    #                    If so, it won't show up in summary.
+    def trigger_action_by_name(method_sym, custom_dir, from_action, *arguments)
```

Comment:
> Sure, although I'd say that this method shouldn't really be called by end users 😅

That's never stopped someone from doing things before 😉 

> Actually, it seems that this is not valid syntax because of the *arguments later 😬

Damn, you're right. I just double checked that you changed all occurences of trigger_action_by_name, so we should be good.